src/main/java/com/thebois/models/world/World.java

```diff
@@ -50,4 +57,56 @@ public Object find() {
         return copy;
     }
 
+    /**
+     * Returns the structures in an arraylist as the interface IStructures.
+     *
+     * @return The list to be returned.
+     */
+    public ArrayList<IStructure> getStructures() {
+        final ArrayList<IStructure> copy = new ArrayList<>();
```

Comment:
Methods should return the most abstract type possible, so in this case you should try returning an `Iterable<IStructure>` (which just allows simple iteration in a foreach loop), and if the consumers of the method require more functionality, an `Collection<IStructure>`. If neither of those are enough, try `List<IStructure>` which should almost always work. Only return the concrete `ArrayList<IStructure>` when there is no other option.

In summary, try to use these in order from most abstract to least:
`Iterable`
`Collection`
`List`
`ArrayList`